python/MRzeroCore/phantom/voxel_grid_phantom.py

```diff
@@ -389,6 +420,7 @@ def resample_multicoil(tensor: torch.Tensor) -> torch.Tensor:
             resample_multicoil(self.B1),
             resample_multicoil(self.coil_sens),
             self.size.clone(),
+            Masks= resample_masks(self.Masks)
         )
 
     def plot(self) -> None:

```

Comment:
I would prefer an argument `plot_masks=False` that ensures that plotting by default behaves as before

Reply:
Done